scripts/solution_view.py

```diff
@@ -120,10 +144,8 @@ def _pos_llh_callback(self, data):
   def update_table(self):
     self._table_list = self.table.items()
 
-  def pos_llh_callback(self, data):
-    soln = sbp_messages.PosLLH(data)
-    self.pos_table = []
-
+  def pos_llh_callback_spp(self,soln,psuedo_absolutes):
```

Comment:
also, the same function is called for RTK or SPP so maybe drop the `_spp` from the name